### PR TITLE
Tm os integration

### PR DESCRIPTION
**Pull request by zuberikfir**

### CODE REVIEW

File: src/utils/app-utils.ts [hunk 1]
```diff
@@ -14,11 +14,20 @@ export const defaultAppStatus = {
...
+    }, time),
+  );
+
 export async function appInitiator(): Promise<IAppStatus> {
   const redirect_uri = walkme.getRedirectURI();
   const appStatus = { ...defaultAppStatus };
 
   try {
+    await wait();
+
```
Comment (dvira) [inline src/utils/app-utils.ts:30]: Not sure what was your intention with this, care to explain?

**Review by dvira — CHANGES_REQUESTED** [review]

File: src/hooks/useShellOS.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      getSellOSClient().onEnvironmentChanged(async ({ data: { Id } }: any) => {
+        setAppEnvironment({
+          dispatch,
+          environments,
+          envId: parseInt(Id as string, 10),
+        });
+
+        return true;
+      });
+  }, [environments]);
```
[the remark this answers is not in the document]
Comment (zuberikfir) [reply src/hooks/useShellOS.ts:44]: removed

File: src/images/os/course_icon.svg [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        <g id="WalkMe-OS-Frame-10" transform="translate(-16.000000, -100.000000)">
+            <g id="side-nav" transform="translate(0.000000, 56.000000)">
+                <g id="course-icon" transform="translate(16.000000, 44.000000)">
+                    <circle id="Oval" cx="24" cy="24" r="24"></circle>
+                    <path d="M32,23.034 L32,29.5462532 L31.9186733,29.5472031 C31.363191,31.242887 28.0295012,32.5462532 24,32.5462532 C19.9704988,32.5462532 16.636809,31.242887 16.0813267,29.5472031 L16,29.5462532 L16,23.034 L24.0000972,27 L32,23.034 Z M24,15 L36,21.0507969 L24.0000972,27 L12,21.0507969 L24,15 Z" id="cuurses-icon" stroke="#C4CFF9" stroke-width="2" stroke-linejoin="round"></path>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
[the remark this answers is not in the document]
Comment (zuberikfir) [reply src/images/os/course_icon.svg:14]: yes. we need them for the OS side bar links

File: src/utils/app-utils.ts (hunk 1 above)
Comment (zuberikfir) [reply src/utils/app-utils.ts:30]: removed

**Review by dvira — APPROVED** [review]
NOICE!

**Pull request merged**